src/Libraries/CoreNodeModels/Input/ColorPalette.cs

```diff
@@ -107,6 +108,9 @@ protected override bool UpdateValueCore(UpdateValueParams updateValueParams)
                 case "DsColor":
                     this.DsColor = this.DeserializeValue(value);
                     return true;
+                case "InputValue":
+                    this.DsColor = this.DeserializeValue(value);
```

Comment:
won't this fail as the formats are different unless you also update this method?

Reply:
right now I haven't updated the individual color property names, just DsColor to InputValue.  I was waiting to hear back from Greg but we definitely need to use InputValue.

Reply:
The only services that TS has right now that use color use the format in the schema.  Where does the `GeometryColor.ByGeometryColor` node come from?

Reply:
Dynamo/Libraries/GeometryColor. It's a function that takes geometry and a color, and returns a geometry with vertex colors set to that color.

Reply:
https://github.com/DynamoDS/Dynamo/blob/master/src/Libraries/CoreNodes/Color.cs

Reply:
We should probably also fix the CoGS output to use the schema `Color` type.

Reply:
That seems like a good idea, I'm not sure I understand your second point though.

Reply:
Right now if you execute a graph with a ColorInputNode, the return type is `Autodesk.Thunderstorm.color`, it should really have the type `Color` from the swagger schema.

Reply:
hmm, so then all TS functions will accept swagger color or TS color?

Reply:
I'm just looking at what CoGS returns to the caller as the value of a node whose `InputValue` is `Color`.  It would be nice if they matched.

Reply:
I'm still trying to figure out if it's possible.  It may be a stringified value.